diffrax/_brownian/tree.py

```diff
@@ -261,27 +262,23 @@ def _evaluate_leaf(
         key,
         r: RealScalarLike,
         struct: jax.ShapeDtypeStruct,
-    ) -> LevyVal:
+    ) -> _CompatibleLevyArea:
         shape, dtype = struct.shape, struct.dtype
 
         t0 = jnp.zeros((), dtype)
         r = jnp.asarray(r, dtype)
 
-        if self.levy_area == "space-time":
-            state_key, init_key_w, init_key_la = jr.split(key, 3)
-            bhh_1 = jr.normal(init_key_la, shape, dtype) / math.sqrt(12)
-            bhh_0 = jnp.zeros_like(bhh_1)
-            bhh = (bhh_0, bhh_1, bhh_1)
-            bkk = None
-
-        else:
-            state_key, init_key_w = jr.split(key, 2)
-            bhh = None
-            bkk = None
-
+        state_key, init_key_w, init_key_la = jr.split(key, 3)
```

Comment:
This was originally inside the `if self.levy_area == "space time"` branch so that we wouldn't generate an unused `init_key_la` in the common case that the output doesn't require Levy area. I think we should keep that, for efficiency.